test/controllers/RemoveMemberDetailsControllerSpec.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import controllers.RemoveMemberDetailsController.{form, viewModel}
+import eu.timepit.refined.{refineMV, refineV}
+import forms.YesNoPageFormProvider
+import models.{CheckMode, NameDOB, NormalMode}
+import pages.{AccountingPeriodPage, MemberDetailsPage}
+import views.html.YesNoPageView
+
+import java.time.LocalDate
+
+class RemoveMemberDetailsControllerSpec extends ControllerBaseSpec {
```

Comment:
You need to add tests here that assert that the value has been removed from the cache